quizbizz/src/util/toasts/toasts.css

```diff
@@ -0,0 +1,18 @@
+.toasts-container{
+    z-index: 1000;
```

Comment:
Suggestion use variables for z-index, it will help you keep record in bigger projects

Reply:
Done